spec/request/appeals_request_spec.rb

```diff
@@ -32,7 +32,7 @@
       let(:user) { FactoryBot.create(:user, :loa1, ssn: '111223333') }
 
       it 'returns a forbidden error' do
-        get '/v0/appeals', nil, 'Authorization' => "Token token=#{session.token}"
```

Comment:
Does this mean we aren't testing the `/v0/appeals` endpoint now?

Reply:
It's covered [now](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/request/appeals_request_spec.rb#L13-L16), this is a typo for v2 the authorization (although as they're in the same controller the before_action applies to both)